src/test/codegen/abi-efiapi.rs

```diff
@@ -22,6 +24,8 @@ trait Copy { }
 
 //x86_64: define win64cc void @has_efiapi
 //i686: define void @has_efiapi
+//aarch64: define void @has_efiapi
 //arm: define void @has_efiapi
+//riscv: define void @has_efiapi
```

Comment:
You need to use `CHECK` for this, right?

Reply:
Maybe. I simply reverted https://github.com/rust-lang/rust/pull/66084 here and didn't make any changes.